src/main.js

```diff
@@ -232,7 +234,7 @@ $(document).ready(() => {
       let timerId = null;
 
       const startTimer = (delay) => {
-        if (!isMouseInSidebar && !isSidebarPinned()) {
+        if (this.store.get(STORE.HOVEROPEN) && !isMouseInSidebar && !isSidebarPinned()) {
```

Comment:
Not sure about this check. If not hover-open, meaning users might not know how to close. I think we should still close automatically in float mode.

Reply:
If user prefer to clicking instead of hovering, I think clicking to close would make more sense. Users will intuitively know it because the behaviour is similar to other apps.

Reply:
I agree. The only probably is we hide the toggle, so at least initially users don't know they need to click in the page to close. Not sure if it could confuse first time users and cause uninstallation.

Reply:
for initial users, they use hovering as default so the timer still works.

Reply:
I still think it's confusing that the behavior is very implicit. Let's not implement it for now until we're clear this is desirable.